OpenDreamShared/Dream/IconAppearance.cs

```diff
@@ -144,7 +144,8 @@ public enum AppearanceProperty {
                     color += "ff";
                 }
 
-                Color = Convert.ToUInt32(color, 16);
+                Color = uint.Parse(color, System.Globalization.NumberStyles.HexNumber);
```

Comment:
Someone tell me if this is actually the same result or not.

Reply:
`Convert.ToUInt32` accepts null while `uint.Parse` instead throws an exception. That wouldn't cause any issues here.